scripts/docs/c4-component.puml

```diff
@@ -3,33 +3,33 @@
 HIDE_STEREOTYPE()
 
 Boundary(chain_db, "Chain DB") {
-        ComponentDb(immutable_db, "Immutable DB", "FS", "Stores immutable blocks", $link="https://input-output-hk.github.io/ouroboros-network/ouroboros-consensus/Ouroboros-Consensus-Storage-ImmutableDB.html")
-        ComponentDb(volatile_db, "Volatile DB", "FS", "Stores new blocks", $link="https://input-output-hk.github.io/ouroboros-network/ouroboros-consensus/Ouroboros-Consensus-Storage-VolatileDB.html")
-        ComponentDb(ledger_db, "Ledger DB", "FS", "Stores ledger state at tip of chain", $link="https://input-output-hk.github.io/ouroboros-network/ouroboros-consensus/Ouroboros-Consensus-Storage-LedgerDB.html")
+        ComponentDb(immutable_db, "Immutable DB", "FS", "Stores immutable blocks", $link="https://input-output-hk.github.io/ouroboros-consensus/haddocks/ouroboros-consensus/Ouroboros-Consensus-Storage-ImmutableDB.html")
+        ComponentDb(volatile_db, "Volatile DB", "FS", "Stores new blocks", $link="https://input-output-hk.github.io/ouroboros-consensus/haddocks/ouroboros-consensus/Ouroboros-Consensus-Storage-VolatileDB.html")
+        ComponentDb(ledger_db, "Ledger DB", "FS", "Stores ledger state at tip of chain", $link="https://input-output-hk.github.io/ouroboros-consensus/haddocks/ouroboros-consensus/Ouroboros-Consensus-Storage-LedgerDB.html")
 }
 
 
-Component(mempool, "Mempool", "", "Holds new transactions past the current tip of the chain", $link="https://input-output-hk.github.io/ouroboros-network/ouroboros-consensus/Ouroboros-Consensus-Mempool.html")
-Component(syncer, "Mempool Syncer", "", "Synchronizes the mempool with latest state of the chain", $link="https://input-output-hk.github.io/ouroboros-network/ouroboros-consensus/Ouroboros-Consensus-Mempool-Impl.html")
-Component(block_creation, "Block Creator", "", "Creates new block out of txs in the mempool", $link = "https://input-output-hk.github.io/ouroboros-network/ouroboros-consensus/Ouroboros-Consensus-Block-Forging.html")
+Component(mempool, "Mempool", "", "Holds new transactions past the current tip of the chain", $link="https://input-output-hk.github.io/ouroboros-consensus/haddocks/ouroboros-consensus/Ouroboros-Consensus-Mempool.html")
+Component(syncer, "Mempool Syncer", "", "Synchronizes the mempool with latest state of the chain", $link="https://input-output-hk.github.io/ouroboros-consensus/haddocks/ouroboros-consensus/Ouroboros-Consensus-Mempool-Impl.html")
```

Comment:
This is a 404, I think is closest to what we want (as opening a mempool will fork the syncing background thread):
```suggestion
Component(syncer, "Mempool Syncer", "", "Synchronizes the mempool with latest state of the chain", $link="https://input-output-hk.github.io/ouroboros-consensus/haddocks/ouroboros-consensus/Ouroboros-Consensus-Mempool-Init.html")
```

Reply:
thanks! I checked most but not all of the links to see if they went somewhere lol